src/Commands/GenerateSitemap.php

```diff
@@ -18,7 +18,7 @@ public function handle(): int
         Eventy::action('rapidez.sitemap.generate');
 
         foreach (Rapidez::getStores() ?: [] as $store) {
-            GenerateSitemapJob::dispatch($store['store_id']);
+            GenerateSitemapJob::dispatchSync($store['store_id']);
```

Comment:
Do we want this running synchronously? It will slow down generation with multiple stores

Reply:
Yes this was intended, i was running into some problems when putting this onto a queue where it would not actually generate the sitemap.
Figured that we might as well just do it like this, since it's only ran through the command anyways.